modules/core/src/main/java/org/apache/ignite/internal/processors/cache/CacheConfigurationSplitterImpl.java

```diff
@@ -105,6 +115,27 @@ public CacheConfigurationSplitterImpl(Marshaller marshaller) {
         return new CacheConfigurationEnrichment(enrichment, fieldClsNames);
     }
 
+    /**
+     *  Checks the field on support of splitter.
+     *
+     * @param field Field.
+     * @return True if splitter serialization is supported, false otherwise.
+     */
+    private boolean isDplitterSupport(Field field) {
```

Comment:
Let's just change it to `supports`. What do you think?

Reply:
Ok.

Reply:
I mean `private boolean supports(Field field) {`